src/main/java/com/flightstats/hub/app/ClusterHubBindings.java

```diff
@@ -28,6 +25,22 @@
 class ClusterHubBindings extends AbstractModule {
     private final static Logger logger = LoggerFactory.getLogger(ClusterHubBindings.class);
 
+    @Inject
```

Comment:
Were these automatically moved? I like having the `@Override`/`configure()` method first since I think it's the one special sort of method in this file and the others are individual dependency specifications.